pkg/util/loop/loop.go

```diff
@@ -77,11 +77,30 @@ type Info64 struct {
 	Init           [2]uint64
 }
 
+func (loop *Device) AttachFromFile(image *os.File, mode int, number *int) error {
+	maxRetries := 5
+	var err error = nil
+
+	for i := 0; i < maxRetries; i++ {
+		err = loop.attachFromFile(image, mode, number)
```

Comment:
What is the motivation to no longer sleep here? Presumably the 250ms sleep * 5 attempts was originally motivated by some situation in which calls can transiently fail with EAGAIN and subsequently work after some time has passed? 

I guess the thinking is probably that since the retry loop body (in `attachFromFile`) now loops through all of the loop devices instead of just a single one, the resulting delay between successive attempts on any one device would be longer... but do you have a sense in practice if that is anything close to on the order of 250ms?

Reply:
Looks like I forgot to put the sleep command back in there. Interesting that things work fine without it though. It is indeed probably because more time is now spent looping through all loop devices before returning from `attachFromFile`

I will add it back.